content/docs/faq-structure.md

```diff
@@ -8,18 +8,67 @@ category: FAQ
 
 ### Is there a recommended way to structure React projects?
 
+React doesn't have opinions on how you put files into folders. That said there are a few common approaches popular in the ecosystem you may want to consider.
+
+#### Grouping by features or routes
+
 One common way to structure projects is locate CSS, JS, and tests together inside folders grouped by feature or route.
 
 ```
-FeatureA
+common/
+  Avatar.js
```

Comment:
Any chance I could sway things from PascalCase to kebab-case? I've seen many people (especially beginners) have issues with [case-sensitivity across operating systems](http://www.informit.com/articles/article.aspx?p=1144082&seqNum=4) and [with git](https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git). I've found it best to avoid these issues altogether by encouraging all lower case and using kebab-case to separate words. I'd love it if the recommended approach in React's docs helped encourage this as well to avoid casing issues for people following the suggestion.

Reply:
I think that's going to be a hard sell for most people since it doesn't match component names.

Reply:
Interesting! I've haven't ran into case-sensitivity issues in my own development yet, but this does sound like a very real concern. I would support using kebab-case, and maybe even explaining why it is recommended over PascalCase.

Reply:
I think kebab-case is a little less intuitive on how to import without opening the file up, i.e.
is `my-component` imported as `MyComponent`, `myComponent`, or `mycomponent`

I think if you maintain the casing and the component name to match, it makes it easier to not constantly reference the file.

Reply:
Kebab-case is for npm packages. PascalCase for local components. My 0.02€.

Reply:
I fairly regularly see Gatsby issues where people can't deploy from linux CI because of casing issues not discovered on their mac.

Reply:
@kentcdodds One viable alternative you could do is encourage ES Lint.

We won't get into the savagery of `eslint-config-airbnb`, but I think you could recommend `import/no-unresolved` which throws error when the import declaration doesn't match the underlying filesystem.